Hse fix

jinzhta opened the pull request:
Re-Cmake tp construct GNU dependency @jinzhta

Add Omega Feature in Running Cider

Final Working Version before Making it faster 2. Fixed datetime problem and and Commented out some useless codes

Comment out some lines because 'MappedXC' object has no attribute 'desc_version'

kylebystrom commented:
Thanks @jinzhta ! A few things that should be changed:
- It looks like you committed some files that are not meant to be part of the code base. (This was an error on my end; I forgot to add these to the .gitignore). Please `git rm` all the new files that weren't in the repo before, commit, then "squash" the commits so those files don't end up in the git history (https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together).
- All the debug-related comments/print statements should be removed in the transform data module.
- Please add a unit test for the new feature in `ciderpress.dft.tests.test_transform_data` to check the derivatives against finite difference. You can follow the other examples in that test module.

jinzhta committed:
Re-Cmake and add Omega Feature (in running Cider for hybrid DFT calculations) with cleanup

    Original commits:
    [846278f] Re-Cmake tp construct GNU dependency
    [fa0977c] Add Omega Feature in Running Cider
    [5a7ae3e] Final Working Version before Making it faster 2. Fixed datetime problem and and Commented out some useless codes
    [c8c9650] Comment out some lines because 'MappedXC' object has no attribute 'desc_version'

    Major changes:
    - Re-Cmake to construct GNU dependency, update CMakeLists.txt for proper dependency setup
    - Add Omega Feature in Running Cider for hybrid DFT calculations
    - Fix datetime problem in data processing
    - Clean up unused code and remove redundant debug statements
    - Fix some problem
    - Add Omegmap Unitest

    Technical fixes:
    - Fix MappedXC object attribute issue for 'desc_version'
    - Update data normalization in transform_data.py
    - Modify cider_paw.py for improved data handling

    Build and maintenance:
    - Update .gitignore to properly exclude build files and dependencies
    - Clean up repository structure by removing tracked build artifacts

jinzhta committed:
Remove explanatory comments and print statements for data statistics in OmegaMap class
